res/values-sq/strings.xml

```diff
@@ -871,222 +869,222 @@
     <string name="TEAM_DEATHMATCH">"Ekipi Deathmatch"</string>
     <string name="Speed_Click_Toggle">"Speed ​​Kliko Toggle"</string>
     <string name="MOD">"MOD"</string>
-    <string name="FAQ">Frequently Asked Questions</string>
-    <string name="BUY_EXTRA_SPINS">BUY EXTRA SPINS</string>
-    <string name="Spins_Remaining">Spins Remaining</string>
-    <string name="One_Spin">One Spin (1)</string>
-    <string name="Bundle_of_Spins">Bundle Of Spins (10)</string>
-    <string name="Plethora_of_Spins">Plethora Of Spins (50)</string>
-    <string name="SIGN_OUT_ALL_DEVICES">SIGN OUT ALL DEVICES</string>
-    <string name="Tournament_Forming">Tournament Forming</string>
-    <string name="TeamTourneyDescription">You and your teammate must register to play in the team tournament. You will both be charged the plasma when the tournament starts but you can unregister before it starts. Needs at least %d contestants to start.</string>
-    <string name="Free_Agent">Free Agent</string>
-    <string name="This_player_selected_a_different_teammate_">This player selected a different teammate.</string>
-    <string name="muted_until">Muted until</string>
-    <string name="Content_rating_of_the_app_may_change_when_playing_multiplayer">Content rating of the app may change when playing multiplayer.</string>
+    <string name="FAQ">"FAQ"</string>
+    <string name="BUY_EXTRA_SPINS">"BLINI SPINA SHTESË"</string>
+    <string name="Spins_Remaining">"Rrotullimet e mbetura"</string>
+    <string name="One_Spin">"Një rrotullim (1)"</string>
+    <string name="Bundle_of_Spins">"Pakoja e rrotullimeve (10)"</string>
+    <string name="Plethora_of_Spins">"Bollëk rrotullimesh (50)"</string>
+    <string name="SIGN_OUT_ALL_DEVICES">"DIL TË GJITHA PAJISJET"</string>
+    <string name="Tournament_Forming">"Formimi i Turneut"</string>
+    <string name="TeamTourneyDescription">"Ju dhe shoku juaj i skuadrës duhet të regjistroheni për të luajtur në turneun e ekipit. Ju të dy do të tarifoheni me plazmën kur të fillojë turneu, por mund të çregjistroheni përpara se të fillojë. Duhen të paktën %d garues për të filluar."</string>
+    <string name="Free_Agent">"Agjent falas"</string>
+    <string name="This_player_selected_a_different_teammate_">"Ky lojtar zgjodhi një shok tjetër skuadre."</string>
+    <string name="muted_until">"I heshtur deri në"</string>
+    <string name="Content_rating_of_the_app_may_change_when_playing_multiplayer">"Vlerësimi i përmbajtjes së aplikacionit mund të ndryshojë kur luani me shumë lojtarë."</string>
```

Comment:
Lines from 872 to 884:
The quotation marks " aren't needed and actually they are useless practically. Please remove them.